tasks/configuration.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+- name: ensure rsyslogd udp reception not commented/disabled
+  lineinfile:
+    dest: '{{ rsyslog_cfg }}'
+    regexp: '\\$ModLoad imudp'
+    line: '$ModLoad imudp'
```

Comment:
Please don't do this by default. I suggest to define a defaults variable which is set to `false` by default. Then you can overwrite it and set it to `true` if required.